build/config/compiler/BUILD.gn

```diff
@@ -26,7 +26,7 @@ if (current_os == "mac" || current_os == "ios") {
 declare_args() {
   # Enable -Werror. This can be disabled if using a different compiler
   # with unfixed or unsupported wanings.
-  treat_warnings_as_errors = true
+  treat_warnings_as_errors = false
```

Comment:
Please make that as true and check the build.

Reply:
Changes added